docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md

```diff
@@ -0,0 +1,336 @@
+# ADB - Android Debug Bridge
+
+Requirements: 
+
+ * [ADB Addon](https://github.com/hassio-addons/addon-adb/blob/v0.6.1/README.md)
```

Comment:
This should probably link to the entire repo directly

```suggestion
 * [ADB Addon](https://github.com/hassio-addons/addon-adb#readme)
```

Reply:
Changed URL to link to `https://github.com/hassio-addons/addon-adb/`